torch/distributed/fsdp/flat_param.py

```diff
@@ -63,6 +65,9 @@
 or a submodule chosen by the provided wrapping policy.
 """
 
+# Environment variable to use unsafe `setattr()` for view setting
```

Comment:
Can we have some description why we need to switch between UNSAFE and SAFER version?

Reply:
Good point! Added an explanation.